leafsmart-front-end/src/components/Favourites.js

```diff
@@ -1,41 +1,31 @@
 import React, { useEffect, useState } from "react";
-import axios from "axios";
 import FavouritesCard from "./FavouritesCard";
 import { useAuth } from "../contexts/AuthContext";
 import { MdFavorite } from "react-icons/md";
 
 const Favourites = () => {
-  const [favourites, setFavourites] = useState([]);
+  const [favs, setFavs] = useState([]);
   const [open, setOpen] = useState(false);
   const [user, setUser] = useState({});
 
-  const { currentUser } = useAuth();
+  const { currentUser, favourites, returnFavourites } = useAuth();
 
   useEffect(() => {
     setUser(currentUser);
   }, [currentUser]);
 
   const handleClick = async (e) => {
     e.preventDefault();
-    axios
-      .post("/favourites", {
-        user_id: user.id,
-      })
-      .then((res) => {
-        const data = res.data;
-        setFavourites(data);
-        setOpen(!open);
-      })
-      .catch((err) => {
-        console.log(err);
-      });
+    returnFavourites()
+    setFavs(favourites)
+    setOpen(!open);
   };
 
-  const favArr = favourites.map((favs) => {
+  const favArr = favs.map((favs) => {
     return <FavouritesCard key={favs.geoname_id} name={favs.city_name} />;
   });
 
-  const SideBarIcon = ({ icon, text = 'tooltip 💡' }) => (
+  const SideBarIcon = ({ icon }) => (
```

Comment:
SideBarIcon is defined in three places -- in the dashboard layout I refactored it to be its own component so you wouldn't have to maintain in three places! Would you like to incorporate that change before merging it in?

Reply:
It shouldn't affect much of the changes I did here since I was just removing the tooltip bit so if there is a conflict on merge I'd maintain the changes you did instead.

Reply:
When I pulled down the branch just now, we've got two spots where a SideBarIcon component is defined (Login and Logout.js) in addition a a seperate component file holding them, they sure aren't breaking anything, but I think it'd be wise to clean 'em up so we don't accidentally use the wrong function!